test/metric_value_benchmark/container_insights_test.go

```diff
@@ -6,26 +6,37 @@
 package metric_value_benchmark
 
 import (
+	_ "embed"
+	"fmt"
+	"github.com/aws/amazon-cloudwatch-agent-test/environment"
+	"github.com/aws/amazon-cloudwatch-agent-test/internal/awsservice"
 	"github.com/aws/amazon-cloudwatch-agent-test/test/metric"
 	"github.com/aws/amazon-cloudwatch-agent-test/test/metric/dimension"
 	"github.com/aws/amazon-cloudwatch-agent-test/test/status"
 	"github.com/aws/amazon-cloudwatch-agent-test/test/test_runner"
+	"github.com/qri-io/jsonschema"
+	"strings"
 	"time"
 )
 
 type ContainerInsightsTestRunner struct {
 	test_runner.BaseTestRunner
 }
 
+//go:embed agent_resources/container_insights_node_telemetry.json
+var emfContainerInsightsSchema string
+
 var _ IECSTestRunner = (*ContainerInsightsTestRunner)(nil)
 
-func (t *ContainerInsightsTestRunner) validate() status.TestGroupResult {
+func (t *ContainerInsightsTestRunner) validate(e *environment.MetaData) status.TestGroupResult {
```

Comment:
Seems it quite become more complex than I expect. But its not worth to solve it right away.

Reply:
What is complex about this? I am passing in the environment so I can get the cluster name and ARN